suites/elasticTest.js

```diff
@@ -0,0 +1,21 @@
+Feature('es');
+
+const { expect } = require('chai');
+const { search } = require('../utils/elasticsearch.js');
+
+const index = 'etl_mickey_1';
```

Comment:
Was that supposed to be hardcoded?

Reply:
Again, sample test ... once we start using it, we can remove this